docs/guides/automations.md

```diff
@@ -0,0 +1,349 @@
+        num-=1
+    logger.info(f"Built {copy} names: {df}")
+    return df
+
+if __name__ == "__main__":
+    list_of_names = build_names()
+```
+
+From here, we can see that the data cleaning workflow has visibility into each step, and we are sending a list of names to our next step of our pipeline.
+
+## Create notification block within the UI
+Now let's try to send a notification based off a completed state outcome. We can configure a notification to be sent so that we know when to look into our workflow logic. 
+
+1. Prior to creating the automation, let's confirm the notification location. We have to create a notification block to help define where the notification will be sent. 
+![List of available blocks](/img/guides/block-list.png)
+
+2. Let's navigate to the blocks page on the UI, and click into creating an email notification block. 
+![Creating a notification block in the Cloud UI](/img/guides/notification-block.png)
+
+3. Now that we created a notification block, we can go to the automations page to create our first automation.
+![Automations page](/img/guides/automation-list.png)
+
+4. Next we try to find the trigger type, in this case let's do a flow completion. 
+![Trigger type](/img/guides/automation-triggers.png)
+
+5. Finally, let's create the actions that will be done once the triggered is hit. In this case, let's create a notification to be sent out to showcase the completion. 
+![Notification block in automation](/img/guides/notify-auto-block.png)
+
+6. Now the automation is ready to be triggered from a flow run completion. Let's locally run the file and see that the notification being sent to our inbox after the completion.
+![Final notification](/img/guides/final-automation.png)
+
+!!! Tip "No deployment created"
+        Keep in mind, we did not need to create a deployment to trigger our automation, where a state outcome of a local flow run helped trigger this notification block. We are not tied to creating a full deployment in order to have safe responses to our desired outcomes.
+
+## Event based deployment automation 
+We can create an automation that can kick off a deployment instead of a notification. Let's explore how we can programatically create this automation. We will take advantage of our extensive REST API catalogue to help 'automate' the creation of this automation.  
+
+Additionally, find more information in our [REST API documentation](https://docs.prefect.io/latest/api-ref/rest-api/#interacting-with-the-rest-api) on how to interact with the endpoints further.
+
+Let's have local deployment created where we can kick off some work based on how long a flow is running. For example, if the `build_names` flow is taking too long to execute, we can kick off a deployment of the same flow `build_names` but replace the count value with something less.
+
+By following the deployment steps, we can get started by creating a local prefect.yaml that looks like this for our flow `build_names`
+
+```yaml
```

Comment:
Could you please add a tab that uses deploy? Should just be the name = main block for the script above. Making that the default should cut down on the amount of work the user needs to do.